mayastor/tests/add_child.rs

```diff
@@ -0,0 +1,129 @@
+use mayastor::{
+    bdev::{nexus_create, nexus_lookup, ChildStatus},
+    core::{mayastor_env_stop, MayastorCliArgs, MayastorEnvironment, Reactor},
+};
+use std::{ops::Deref, process::Command};
+
+static NEXUS_NAME: &str = "nexus";
+
+static DISKNAME1: &str = "/tmp/disk1.img";
+static BDEVNAME1: &str = "aio:///tmp/disk1.img?blk_size=512";
+
+static DISKNAME2: &str = "/tmp/disk2.img";
+static BDEVNAME2: &str = "aio:///tmp/disk2.img?blk_size=512";
+
+pub mod common;
+
+fn test_start() {
```

Comment:
We have common code for this in test/common.rs I think its best to use that unless there is a specific reason.

Reply:
Thanks - I've used the common code now.